cellxgene_schema_cli/scripts/readme.md

```diff
@@ -35,28 +34,18 @@ The following gene references are currently stored (to see specific versions ref
 To update these gene references, a developer has to do run the following:
 
 ```bash
-make gene_processing
+make update-gene-references
 ```
 
 This will:
 
 1. Download the reference gene files.
-2. For 10X data, uncompress it, get the GTF files and remove the rest.
-3. At the end the following files will be in this folder:
+2. For 10X data: decompress it, get the GTF files and remove the rest.
+3. At the end the following files will be in the folder:
 
     - `mus_musculus.gtf.gz`
     - `homo_sapiens.gtf.gz`
     - `sars_cov_2.gtf.gz`
     - `ercc.txt`
 
 The reference GTF files for mouse and human are obtained from the 10X reference data. The GTF file for sars‑CoV‑2 is obtained from ENSEMBL. The ERCC IDs are obtained from ThermoFisher.
-To download all these files the following has to be called:
-
-If reference gene versions need to be updated then `Makefile` has to be changed accordingly under the rules corresponding to each organism.
-
-## Cleanup
-
-If instead of the rule `update-references` individual make rules were executed, then make sure to remove all unused downloads using:
-```bash
-make clean
-```
```

Comment:
I judge this to all be outdated/irrelevant. We no longer have make recipes for individual files.